src/Utilities/model.jl

```diff
@@ -318,6 +327,62 @@ function MOI.get(model::AbstractModel, ::MOI.ListOfModelAttributesSet)::Vector{M
 end
 
 # Constraints
+single_variable_flag(::Type{<:MOI.EqualTo}) = 0x1
+single_variable_flag(::Type{<:MOI.GreaterThan}) = 0x2
+single_variable_flag(::Type{<:MOI.LessThan}) = 0x4
+single_variable_flag(::Type{<:MOI.Interval}) = 0x8
+single_variable_flag(::Type{MOI.Integer}) = 0x10
+single_variable_flag(::Type{MOI.ZeroOne}) = 0x20
+single_variable_flag(::Type{MOI.Semicontinuous}) = 0x40
+single_variable_flag(::Type{MOI.Semiinteger}) = 0x80
+
+# Sets setting lower bound:
+lower_bound(set::MOI.EqualTo) = set.value
+lower_bound(set::Union{MOI.GreaterThan, MOI.Interval}) = set.lower
+# 0xb = 0x8 | 0x2 | 0x1
+const LOWER_BOUND_MASK = 0xb
```

Comment:
I think this should include `Semicontinuous` and `Semiinteger`. These sets have bound parameters. Same for `UPPER_BOUND_MASK`.

Also please double check that `Semicontinuous` and `Semiinteger` are covered by tests.